packages/List/src/List.stories.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import Happiness from '@igloo-ui/icons/dist/Happiness';
+
+import readme from '../README.md';
+
+import List, { Option, Member, OptionType } from './List';
+
+export default {
+  title: 'Components/List',
+  component: List,
+  parameters: {
+    description: readme,
+  },
+} as ComponentMeta<typeof List>;
+
+const largeOptionList: Option[] = [
+  {
+    type: 'list',
+    label: 'Item 1',
+    value: '1',
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 2 (disabled)',
+    value: '2',
+    disabled: true,
+  },
+  {
+    type: 'list',
+    label: 'Item 3 (focused)',
+    value: '3',
+    icon: <Happiness size="small" />,
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 4 (selected)',
+    value: '4',
+  },
+  {
+    type: 'list',
+    label: 'Item 5',
+    value: '5',
+    color: '#74DCC9',
+  },
+  {
+    type: 'list',
+    label: 'Item 6',
+    value: '6',
+  },
+  {
+    type: 'list',
+    label: 'Item 7',
+    value: '7',
+  },
+];
+
+const premiumOptionlist: Option[] = [
+  {
+    type: 'list',
+    label: 'Item 1',
+    value: '1',
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 2',
+    value: '2',
+  },
+  {
+    type: 'list',
+    label: 'Item 3 (Premium)',
+    value: '3',
+    icon: <Happiness size="small" />,
+    premium: true,
+  },
+  {
+    type: 'list',
+    label: 'Item 4',
+    value: '4',
+  },
+];
+
+const membersList: Member[] = [
```

Comment:
As for the default list, let's try to give examples close to the use cases.

<img width="636" alt="Capture d’écran, le 2023-02-09 à 13 02 08" src="https://user-images.githubusercontent.com/2379339/217899081-a75c8655-7c2c-4b60-8404-bd108736b06c.png">